alerts/state.libsonnet

```diff
@@ -0,0 +1,37 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'state-alert.rules',
+        rules: [
+          {
+            alert: 'CephClusterErrorState',
+            expr: |||
+              ceph_health_status{%(cephExporterSelector)s} > 1
+            ||| % $._config,
+            'for': $._config.clusterStateAlertTime,
+            labels: {
+              severity: 'critical',
+            },
+            annotations: {
+              message: 'Ceph cluster is in error state',
+            },
+          },
+          {
+            alert: 'CephClusterWarningState',
+            expr: |||
+              ceph_health_status{%(cephExporterSelector)s} == 1
```

Comment:
The rules look ok to me. I have just got a bit a question about what does the `cephExporterSelector` do?

Reply:
This will match the lables for the metrics points. At the moment the label to be matched is `job="ceph"`